Backpack/src/main/res/values/backpack.font.xml

```diff
@@ -1,79 +1,80 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <!-- Text should have no lineSpacingMultiplier to work with relative -->
-  <style name="bpkTextBaseInternal" parent="bpkTextBase">
+
+  <style name="bpkTextBaseRelative" parent="bpkTextBase">
```

Comment:
I am 90% sure that these don't actually don't do anything, but keeping it in case somebody is doing something outside of our components that requires this